crates/vhost-user-backend/src/vring.rs

```diff
@@ -433,6 +460,14 @@ impl<M: 'static + GuestAddressSpace> VringT<M> for VringRwLock<M> {
         self.write_lock().set_queue_next_avail(base)
     }
 
+    fn set_queue_next_used(&self, idx: u16) {
+        self.write_lock().set_queue_next_used(idx)
+    }
+
+    fn queue_used_idx(&self) -> Result<u16, VirtQueError> {
+        self.write_lock().queue_used_idx()
```

Comment:
Is `write_lock` needed here?

Reply:
Opss true, dumb copy-paste error, thanks!